andela/ah-alpha
# #162948898 upgrade django from 1.10 to 2.1.4

#### What does this PR do?
Upgrade Django from version 1.10 to 2.1.4

#### Description of Task(s) to be completed?
* Upgrade Django from v1.10 to v2.1.4
* Upgrade all the other 3rd party packages used in the project to their latest versions
* Remove imports that were available in Django 1.10(`django.conf.urls`) but deprecated in Django 2.1.4, and use updated ones instead(`django.urls`)
* Temporarily comment out JWT Authentication from the list of default authentication classes since it hasn't been implemented yet
* Updated all references to Django v1.1 in the docstrings and comments to Django v2.1.4

#### How should this be manually tested?
* Pull the branch `git fetch origin ch-django-upgrade-162948898:ch-django-upgrade-162948898`
* Checkout to the branch `git checkout ch-django-upgrade-162948898`
* Install requirements `pip3 install -r requirements.txt`
* Check the versions of the installed packages by running `pip3 freeze` in your terminal

#### Any background context you want to provide?
* This project uses the latest version of python3. Please download it from [here](https://www.python.org/downloads/)
* You may may as well use pip instead of pip3 if you have either only python3 installed on your computer or you have it set as the default version of python

#### What are the relevant pivotal tracker stories?
[#162948898](https://www.pivotaltracker.com/story/show/162948898)
